src/chrome/content/rules/heritage.gov.hk.xml

```diff
@@ -0,0 +1,13 @@
+<ruleset name="heritage.gov.hk">
```

Comment:
Note as `Invalid certificate` in a [top comment](https://github.com/EFForg/https-everywhere/search?utf8=%E2%9C%93&q=problematic):

heritage.gov.hk